src/asm/lexer.c

```diff
@@ -354,6 +354,7 @@ struct LexerState {
 	uint32_t colNo;
 	int lastToken;
 	int nextToken;
+	bool isAtEOF;
```

Comment:
Isn't this equivalent to `lastToken == EOB`?

Reply:
No, because `T_EOB` can occur throughout a file at the end of `REPT`s/`FOR`s/`INCLUDE`s/macro invocations. Only the very last one is a real EOF that should stop the parser by sending `T_EOF`.